tracer/src/Datadog.Trace/Sampling/OverheadController.cs

```diff
@@ -64,5 +64,5 @@ public void Reset()
         }
     }
 
-    private static int ComputeSamplingParameter(int pct) => (int)Math.Round(100m / pct);
+    private static int ComputeSamplingParameter(int pct) => pct == 0 ? pct : (int)Math.Round(100m / pct);
```

Comment:
Nit: since it's a signed int, should we check for negative values?

Reply:
fixed in [9d5ab57](https://github.com/DataDog/dd-trace-dotnet/pull/4975/commits/9d5ab579acb99322092a7850a570b3eae20878fe)